src/routes/index.js

```diff
@@ -1,9 +1,11 @@
 import { Router } from 'express';
 import authRoute from './auth';
+import requestRoute from './request';
 
 const router = Router();
 
 
 router.use('/auth', authRoute);
+router.use(requestRoute);
```

Comment:
Please rename the name of the route, instead of `requestRoute`, you can use `/requests`